test/test/pytools/test_package_version.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import re
+from urllib import request
+from xml.etree import ElementTree
+
+import pytools
+
+log = logging.getLogger(__name__)
+
+
+def test_package_version():
+    dev_version = pytools.__version__
+
+    log.info(f"Test package version – version set to: {dev_version}")
+    assert re.match(
+        r"^(\d)+\.(\d)+\.(\d)+(rc\d)?$", dev_version
```

Comment:
Not that we will need this often, but might want to have two-digit release candidates.

```suggestion
        r"^(\d)+\.(\d)+\.(\d)+(rc\d+)?$", dev_version
```

Reply:
Let's do, I considered it first, too :)